CONTRIBUTING.md

```diff
@@ -40,29 +40,48 @@
 
 ## Coding Conventions
 
-This project uses [flake8](https://flake8.pycqa.org/en/latest/) to ensure basic code sanity and [black](https://github.com/psf/black)
+This project uses [ruff](https://github.com/charliermarsh/ruff) to ensure basic code sanity and [black](https://github.com/psf/black)
 for no-nonsense, consistent formatting.
 
-Both `flake8` and auto-fixing `black` are part of the pre-commit hook that is automatically set up during `make dev`.
+Both `ruff` and auto-fixing `black` are part of the pre-commit hook that is automatically set up during `make dev`.
 
 You can also run the lint and formatter manually:
 
--  To run flake8 run: `make lint`
+-  To run `ruff` run: `make lint`
 -  To reformat code black run: `make format-fix`
 
-**NOTE** If the pre-commit hook finds and auto-corrects some formatting errors, it will not auto-stage
+**NOTE:** If the pre-commit hook finds and auto-corrects some formatting errors, it will not auto-stage
 the updated files and will fail the commit operation. You have to re-drive the commit. This is a well-known and
 unlikely-to-change behavior of the [pre-commit](https://github.com/pre-commit/pre-commit/issues/806) package that this repository uses to manage hooks.
 
-The project documents code by docstrings in rst format. Compliance with rst format may be checked by running `make docs`.
+The project documents code by docstrings in [google-like](https://google.github.io/styleguide/pyguide.html#3-python-style-rules) format.
+
+The project documentation is done in hugo. To contribute:
+
+1. Install [Hugo](https://gohugo.io/)
+
+2. Initialize required submodules
+
+  `git submodule update --init --recursive --depth 1`
+
+3. Install npm dependencies
+
+  `cd docs && npm install`
+
+4. Run the hugo locally
+
+  `hugo server`
```

Comment:
Does this work in root dir or do you need to change directory to docs?

Reply:
FYI user can use `make new-docs`, but there is an issue with path to docs which was not changed. Please, change it as a part of this PR.

Reply:
Please, also mention that documentation for pandas and fdw is still in old-fashioned way, but it will be moved as well.

Reply:
Make also a note that the documentation is published using manually triggered GitHub workflow.